demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -96,6 +96,19 @@ def ignored_errors(self) -> list:
         except:  # noqa: E722
             return []
 
+    def ignored_errors_related_files(self, file_path: str) -> list:
+        try:
+            return (
+                list(
+                    self.in_pack.ignored_errors_dict.get(  # type: ignore
```

Comment:
maybe we should add a class of `PackIgnore` similar to this? https://github.com/demisto/demisto-sdk/pull/3592

would be more readable IMO, and in some cases you would need the `.pack-ignore` not only for validations.